test/lib/tasks/clean_published_repo_directories_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'katello_test_helper'
+require 'rake'
+
+module Katello
+  class CleanPublishedRepoDirectoriesTest < ActiveSupport::TestCase
+    def setup
+      Rake.application.rake_require 'katello/tasks/clean_published_repo_directories'
+      Rake.application.rake_require 'katello/tasks/repository'
+
+      Rake::Task['katello:regenerate_repo_metadata'].reenable
```

Comment:
Ah you're correct. I was getting some test failures but this isn't the line I used to fix it.